@harrah I get the same error message as the failed jenkins build. Is your checkfile up to date?

```
qbin/scalac test/files/neg/t7519-b/Mac_1.scala && qbin/scalac test/files/neg/t7519-b/Use_2.scala
test/files/neg/t7519-b/Use_2.scala:6: error: type mismatch;
 found   : String
 required: Q
  val x: Q = ex.Mac.mac("asdf")
                       ^
one error found
```

--- a/test/files/neg/t7519-b.check
+++ b/test/files/neg/t7519-b.check
@@ -1,0 +1,6 @@
+Use_2.scala:6: error: type mismatch;
+ found   : String
+ required: Q
+  val x: Q = ex.Mac.mac("asdf")
+                       ^
+one error found

--- a/test/files/neg/t7519-b/Mac_1.scala
+++ b/test/files/neg/t7519-b/Mac_1.scala
@@ -1,0 +1,14 @@
+// get expected error message without package declaration
+package ex
+
+import scala.language.experimental.macros
+import scala.reflect.macros._
+
+object IW {
+  def foo(a: String): String = ???
+}
+object Mac {
+  def mac(s: String): String = macro macImpl
+  def macImpl(c: Context)(s: c.Expr[String]): c.Expr[String] = 
+    c.universe.reify(IW.foo(s.splice))
+}

--- a/test/files/neg/t7519-b/Use_2.scala
+++ b/test/files/neg/t7519-b/Use_2.scala
@@ -1,0 +1,8 @@
+trait Q
+trait K
+
+object Use {
+  implicit def cd[T](p: T)(implicit ev: T => K): Q = ???
+  val x: Q = ex.Mac.mac("asdf")
+}
+

